compiler/src/main/java/com/squareup/anvil/compiler/InterfaceMerger.kt

```diff
@@ -64,11 +64,13 @@ internal class InterfaceMerger(
           )
         }
       }
+      // Convert the sequence to a list to avoid iterating it twice. We use the result twice
+      // for replaced classes and the final result.
+      .toList()
```

Comment:
I noticed while debugging that the filter and map lambdas are called twice. That's unnecessary.